src/js/utils/getTimeRemain.js

```diff
@@ -0,0 +1,7 @@
+import { Duration } from "../libs/luxon.js";
```

Comment:
давай лучше испльзовать luxon который установлен через npm, а тот что мы сами в папку копировали - удалим
```suggestion
import { Duration } from "luxon";
```